tests/kernel/common/src/timeout_order.c

```diff
@@ -65,11 +65,12 @@ void test_timeout_order(void)
 	u32_t uptime = k_uptime_get_32();
 
 	/* sync on tick */
-	while (uptime == k_uptime_get_32())
+	while (uptime == k_uptime_get_32()) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```

Comment:
Should this one be after #endif?

Reply:
clang-tidy was not that smart after all :(